python-sdk/src/astro/sql/operators/dataframe.py

```diff
@@ -119,6 +120,7 @@ def __init__(
         database: str | None = None,
         schema: str | None = None,
         columns_names_capitalization: ColumnCapitalization = "original",
+        if_exists: LoadExistStrategy | None = "replace",
```

Comment:
```suggestion
        if_exists: LoadExistStrategy = "replace",
```

Reply:
Wouldn't it break if the user doesn't specify `if_exists` for users in the older version of Astro-SDK?

Reply:
`"replace"` is already there as the default

Reply:
Yes, I tired it locally now. While using annotations, even when the value is not passed at all it picks up the default value.